classes/observation_manager.php

```diff
@@ -19,12 +19,15 @@
  *
  * @package   mod_observation
  * @copyright  2021 Endurer Solutions Team
- * @author Matthew Hilton <mj.hilton@outlook.com>
+ * @author Matthew Hilton <mj.hilton@outlook.com>, Celine Lindeque
  * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
 namespace mod_observation;
 
+use Behat\Behat\Output\Node\Printer\Helper\ResultToStringConverter;
```

Comment:
This include looks like it was auto dragged in by the IDE. Either way, cannot exist in a production system, behat is dev only.